src/app/pages/IdxPage/IdxPage.jsx

```diff
@@ -99,9 +99,9 @@ const IdxPage = ({
         openGraphType="website"
       />
       <LinkedData type="WebPage" seoTitle={seoTitle} />
-      <main role="main">
+      <main role="main" id="content" tabIndex={-1} aria-describedby="headline">
```

Comment:
what is the `aria-describedby="headline"` for?

Reply:
When the screen reader enters `<main role="main" id="content" tabIndex={-1} aria-describedby="headline">` Iw would read what is in `<h1 id="headline">`. I will get rid of this for now :)